### PR TITLE
Improve text of second module

### PR DESCRIPTION
**Pull request by dassencio**

### CODE REVIEW

File: docs/002-basics-of-ci-with-actions.md [hunk 1]
```diff
@@ -2,48 +2,48 @@
 
 ## 1 - Inspect the repository
 
-This repository actually contains a [React](https://reactjs.org/)-application built with [vite](https://vitejs.dev/) that we want to automatically test and build in this part of the workshop.
+This repository contains a [React](https://reactjs.org/)-based application built with [Vite](https://vitejs.dev/). We aim to automate its testing and building in this segment of the workshop.
```
Comment (davelosert) [inline docs/002-basics-of-ci-with-actions.md:5]: I don't like `segment
```suggestion
This repository contains a [React](https://reactjs.org/)-based application built with [Vite](https://vitejs.dev/). We aim to automate its testing and building in this lab.
```

I don't like `segment` for a wording here, an throughout the other parts, we always called it a `lab`, so I'd stay consistent here 🙂

File: docs/002-basics-of-ci-with-actions.md [hunk 2]
```diff
@@ -2,48 +2,48 @@
...
-To test the container, run `docker build . -t local:latest` to build the image and then `docker run -p 8080:8080 local:latest` to run it. Running these commands locally will require you to install [Docker](https://www.docker.com/).
+To test the container, run `docker build . -t local:latest` to build the image and `docker run -p 8080:8080 local:latest` to run it. These commands require a local installation of [Docker](https://www.docker.com/).
 
-## 2 - Add Continuous Integration
+## 2 - Implement Continuous Integration (CI)
 
 ### 2.1 - Use a starter workflow
 
-To develop a GitHub Workflow process that uses Actions to automate the Continuous Integration process, you can begin by adding a **starter workflow** to the repository:
+To build a workflow that employs Actions for automating the Continuous Integration process, start by adding a **starter workflow** to your repository:
```
Comment (davelosert) [inline docs/002-basics-of-ci-with-actions.md:23]:
```suggestion
To build a workflow that employs Actions for your Continuous Integration process, start by adding a **starter workflow** to your repository:
```

The phrase `automating the CI process` doesn't really make sense to me, as CI is something that cant' be really done manually, can it?

File: docs/002-basics-of-ci-with-actions.md [hunk 3]
```diff
@@ -2,48 +2,48 @@
...
 
-1. On the initial view of your repository, find and navigate to the **Actions** tab.
-2. Click `New workflow`
-3. Search for `Node.js`
-4. Click **Configure** on the `Node.js` starter workflow
-5. From the yml-array in the `node-version` field, remove `14.x` (our app is not compatible with this version)
+1. From your repository's main view, find an navigate to the **Actions** tab.
```
Comment (davelosert) [inline docs/002-basics-of-ci-with-actions.md:25]:
```suggestion
1. From your repository's main view, find and navigate to the **Actions** tab.
```

First time I see ChatGPT suggest an actual typo 🙈

File: docs/002-basics-of-ci-with-actions.md [hunk 4]
```diff
@@ -116,40 +116,40 @@ describe("<Home />", (): void => {
...
+The result of your last push to the main branch should resemble the following:
 
-![Actions overview showing a succesful workflow run](./images/success-nodejs-workflow.png)
+![Actions overview showing a successful workflow run](./images/success-nodejs-workflow.png)
 
-## 3 - Add Code Coverage to your workflow
+## 3 - Add code coverage to your workflow
 
-It is common when working on the CI part of your project to add more informations to the user, for example the tests "*code coverage*".
+When setting up CI for your project, it's common to provide additional information to users, such as test "*code coverage*".
```
Comment (davelosert) [inline docs/002-basics-of-ci-with-actions.md:125]:
```suggestion
When setting up CI for your project, it's common to provide additional information to users, such as the test's "*code coverage*".
```

**Review by davelosert — CHANGES_REQUESTED** [review]
@dassencio : Thank you for this. It looks great apart from some minor suggestions I've made! 👍🏻

File: docs/002-basics-of-ci-with-actions.md (hunk 3 above)
Comment (dassencio) [reply docs/002-basics-of-ci-with-actions.md:25]: It's more likely that I introduced the typo myself. Good catch 🙂

File: docs/002-basics-of-ci-with-actions.md (hunk 4 above)
Comment (dassencio) [reply docs/002-basics-of-ci-with-actions.md:125]: I believe the original sentence was correct here. "test's code coverage" implies the code coverage is associated with a single test vs. a set of tests.

Comment (dassencio) [reply docs/002-basics-of-ci-with-actions.md:125]: Possible alternative:

```suggestion
When setting up CI for your project, it's common to provide additional information to users, such as code coverage stats for the project's tests.
```

**Review by davelosert — APPROVED** [review]
Now this looks great! Thank you again @dassencio! ❤️

**Pull request merged**